fellow-submissions/stephanie-smith/style.css

```diff
@@ -0,0 +1,59 @@
+:root {
+  --block-background-color: #92F892;
+}
+
+body {
+  background-color: #92F8F5;
+  font-family: 'Amatic SC', cursive;
+  font-size: 20px;
+}
+
+h1 {
+  text-align: center;
+  letter-spacing: 1.3px;
+}
+
+p {
+  text-align: center;
+  font-size: 25px;
+  font-weight: bold;
+  letter-spacing: 1.3px;
+}
+
+ul {
+    list-style: none;
+}
+
+.block.fixed {
+  margin-bottom: 20px;
+  margin-left: 100px;
+  margin-right: 100px;
+  background-color: #92F892;
+  font-size: 25px;
+  border: 2px solid black;
+  letter-spacing: 1px;
+}
+
+.color.themed {
+  --block-background-color: #92F892;
+  --block-shadow-color: #A992F8;
```

Comment:
I'm impressed and unsurprised that you were able to figure out this CSS framework 😄